pdfminer/layout.py

```diff
@@ -180,8 +191,14 @@ def voverlap(self, obj):
 class LTCurve(LTComponent):
     """A generic Bezier curve"""
 
-    def __init__(self, linewidth, pts, stroke=False, fill=False, evenodd=False,
-                 stroking_color=None, non_stroking_color=None):
+    def __init__(self,
+                 linewidth,
+                 pts,
+                 stroke=False,
+                 fill=False,
+                 evenodd=False,
+                 stroking_color=None,
+                 non_stroking_color=None):
```

Comment:
Don't apply formatting on the code if it is not changing anything

Reply:
My sincerest apologies for this.  I have reverted the change (I didn't catch this change -- I was working from a very stale branch that had a pep8 violation.  I didn't catch that the pep8 had been resolved after I had forked).